app/code/Magento/Ui/Test/Mftf/ActionGroup/AdminOpenColumnActionDropDownForAdminGridActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenColumnActionDropDownForAdminGridActionGroup">
```

Comment:
now that i'm reading your b2b test, i'm seeing that this actiongroup should be deleted. sorry for being wrong before! :(
please delete the file & move the 2 actions into `AdminAddColumnToAdminGridActionGroup` as the first 2 steps